docs/contributor/troubleshooting.md

```diff
@@ -2,6 +2,22 @@
 
 This document contains tips and tricks to common problems with the NATS Manager and will be updated continuously.
 
+## Troubleshooting: Installing NATS Manager using a Docker image
+
+### Error while deploying the NATS Manager
+
+**Symptom:** `make deploy` step fails due to namespace not found
+
+Error message: `Error from server (NotFound): error when creating "STDIN": namespaces kyma-system not found`
+
+**Cause:** The namespace of the deployment does not exist yet.
```

Comment:
```suggestion
**Cause:** The Namespace of the Deployment does not exist yet.
```

[We capitalize Kubernetes resources](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md)